pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -618,10 +652,70 @@ form {
   }
 }
 
+// Carousels
+// Uses the carouFredSel jQuery plugin: https://github.com/nzy/carouFredSel
+.carousel-wrapper {
+  margin: 2em 0 0;
+
+  @media screen and (max-width: $width-sm) {
+    padding-bottom: 1em;
+  }
+
+  // Set a min-height inline on the .carousel div
+  // inside .carousel-wrapper to account for varying
+  // quote lengths and prevent vertical height changes
+  .carousel {
+    overflow: hidden;
+
+    @media screen and (max-width: $width-md) {
+      height: unset !important;
+      overflow: unset;
+    }
+  }
+
+  // The plugin we're using disappears when <html dir="rtl">
+  // - updating the inset value fixes it; it does remove the swiping
+  // animation between slides on desktop, but keeps the content
+  .slides.rtl {
+    inset: 0 !important;
+  }
+
+  p {
+    font-family: $barlowSemiCondensed-medium, sans-serif;
+  }
+
+  .pagination {
+    margin: 1em 0 0 0;
+
+    @media screen and (max-width: $width-sm) {
+      margin-top: 3em;
+    }
+
+    &.left-align {
```

Comment:
I added the class to the carousel on this page in Dropbox beforehand so it would show correctly once this gets merged: https://code.org/educate/professional-learning/middle-high
<img width="1603" alt="Screenshot 2023-06-01 at 3 44 21 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/5ebddd37-df33-48a9-9b49-1fa7c49d0504">

The default on all carousel types before* was `text-align: left` but I wanted to make it optional w/ this class since the photo slideshow used here needs to be centered.  

*see line 717 in the before version the diff

Reply:
Ah I see, thanks for the context!